code/datums/diseases/advance/symptoms/dizzy.dm

```diff
@@ -48,6 +48,6 @@ Bonus
 				to_chat(M, "<span class='warning'>[pick("You feel dizzy.", "Your head spins.")]</span>")
 		else
 			to_chat(M, "<span class='userdanger'>A wave of dizziness washes over you!</span>")
-			M.Dizzy(5)
+			M.dizziness = min(L.dizziness + 30, 100)

```

Comment:
Could potentially reduce dizziness

Reply:
Yeah, I just caught that on my own when I saw one of the tests fail.

The capping of dizziness is intentional, as I've been told that dizziness values above 100 can crash games.